### ISSUE TITLE
[ossia-max] aim.remote does not support clip 

### ISSUE THREAD

**Issue by vincentgoudard** [ossia/libossia#781]
Documentation says it does... but should it ? Is it not rather a parameter attribute ?
So maybe, just the documentation and allowed messages for ossia.remote should be updated...
See patcher below.

<pre><code>
----------begin_max5_patcher----------
694.3oc2W00aaCBE8Y6eEHdZSJKxXG+Q1S8+wTUDwgzPmMXA3VOU0+6iObVx
V5booD0k8PLlavbtm68.W3o3H3Z9.QBAeE7MPTzSwQQVSFCQi8ifs3g5FrzN
LXMuskvTvYt+SQFTV6pcTIXCmHALtB7HW7cvmvMpc7961osW2a9JrhxY.ya5
VIfp9794oSPj6GvpFJiTy6Y1YNa+Hvp5cT1cqDjZkyiyWrXdxLPYg4YVhsIM
YdB31wug02RYMDk00QGLx6U6slLZktwRC956+BpDZr8bbr4wrPDXLwCI.KAj
gNs6S1.+6rBUUdfUn7jPwphygUaa35YdBuMOI05fo4llQuM8s4som5sKO.o.
2RTDwJBCutgbL61xEsXqeTLZwMopezQbdGDNC.WiY2AA2dFrmQdT6LmjR4RI
EOWPZ4JBXMVLU3oH6nvyxpyI7j8BIyD3TL196rXru4ajiWAKeiPWSIbvM0Mz
NvZtZG3FMcAI5F7..85Jgw.WZNJTRgzOZoPxhvJEx92WJ7KOyuk+igHDJOTw
nESmzuLI7JmlEsLn46xqlU9a47opauH8nvyxx.s9t5iZ4sKaixKCZ1N++qr8
X3IXY6hKT1dRBug7.sl.TDoZBFmlZEDYYuC8M5TFOwNYWRcs6nXASVeUUyRq
rOmSvLJ4GODaQUnpk8FJkYgBZte3eboUq6Zr+6QGIuWTue5F2MEbvg2nU7Tl
8RmGMlR2Xdw3uu3T5ANUA.mbOvoH.3T3AN4A.GjG3jE.bx7.GT.vYoO3jDh.
WhGHsLD.4SJxboj2ORo9fTHxRliS+5HEBcGxKg2hSPxswGtq6AhPNNZKH5B.
2yEltUyrcoLWW6gKfBco88i2VFBhE580U5M06EVGCNT3JACa4aHBVO0t6crg
dZHsEWX5BGxNriI1ZPwOG+S.CpVtlB
-----------end_max5_patcher-----------
</code></pre>

**Comment by avilleret** [comment]
by design you can access all the parameter's attributes through remote.
i.e. you can set the clip mode from the remote with a `[clip both(` message.
But setting it via attribute `@clip both` doesn't work and this is bug I'm gonna fix. :-)

see: 

<pre><code>
----------begin_max5_patcher----------
874.3ocyX9saZCCEF+Z3ovJWsIwP1Nw4O6p9dLUUYBtP5H1Q1lVlp569brCT
1pIMrYBbQaHNA97O+cN9bRdc5jnEhcLUD36fe.lL40oSlXGpcfIcmOIpltqb
CUYusnZlRQWwhl4tllsSaGubSUC3QI6vUpVZGWr3ougy2OHeasXqdCSa+wPc
i5FR+qFlalDEAtu6RMTc45J9pGjrRs6pYDxb3L.FG2dffbmLGd36XDohuWCb
6XuMcZ6+lEBFWHzq8wXRPYDhawJAZQLd7PbESCZwzGgjPRXJN+cBSxCOgb1K
lY8G.rQVw0fFpjVyzLoONS8yIrGXRSmSlAPwYsbT.6mFTf8KyOsOLxBocQLD
dQsq+obt3fFQVje79J2HIcnvlzkbDhidVmjUKzdKP.O6Ttr7h1YeLzBQV7sP
JGhD1TN7E1rJE00Lt9CDpWWo.KELEfKzfWDxeB9BcidsX6p0lwK219sn5JAG
z9IyQEnR+UeqHYmswRRrwnYoN60d.CCow1K1szp.TEfsqwLmXK8Q04Wg.km8
NUHBL7T83Fg42vyrsv+jEeXx1UJ7AFmtXC6XPdTHqoVBROYH7LPzBJeUugxI
EN2L1dfbApOdh8cDJUEctaeGvBpux8nSr2SbeDa+q2jWR1QDWjGdhOoeiP2L
FNNK4Ja3f6Nz.A3Nij.n4.cG.4agCeAiD5VJve1yqD1Pg3qen.w8nLH30IT3
.hmJ8Oo+kH+ld+0siOBXDhLhFd102uycA3nhqYl+iBgmUm7KQ9MJAeDvEYin
aStUbajaOtaK2N8B51c.eIb6dAdI64pRFPyTZODGzWAEFibcvL9w0w2Jw0t9
1th0rLQ1mUGLgtVVW3dW+qomQar16HZSE+ue8xVUZG+OWcThsxx8yq8ElAnC
BszDxWwsOz4Q2T66pyrfbuWGXvJY56Cf+LkfAPo1WDN.9IJYmN+2JkLDkRBg
RjQSorQSo3QymPikRHxnoz3wDdHJgBgRwiUrGZPYtwgPI3.TpH.BULDhBw9q
CxjBwJ2.zIDNT5.zgD.cFxVCgnR6P1.OO.5LjxrYePGWKKzllmYRU2Makvz5
1SBY6o4yrmVwcmZarJRZZJe+86FgJMMRoMcQsUZmVQ6Rc8lGUKVxj7sUcsKY
fyHossPtokOUC0wgs6wouM82oXBSbB
-----------end_max5_patcher-----------
</code></pre>

**Comment by vincentgoudard** [comment]
Yes, I was suggesting the same, just updating the doc. Besides, a remote can be bound to multiple parameters with different attributes, so I don't see a clear way of having this accessible in the remote with the current syntax.

I was actually thinking that remote could have its own attributes that adds on top of the parameter's ones. LIke for instance, if I monitor a `ø.parameter @range 0 10 @clip both` with an `ø.remote @range 3 12 @clip both`, the value would eventually be clipped between 3 and 10.
This is just how I thought it would work when I saw that there was a `@clip` attribute to the remote, and I can imagine a few situaton where this could be useful, but I am not meaning I really do want/need this feature.
My 2 cents!
